src/main/java/dev/watchwolf/entities/ServerType.java

```diff
@@ -1,6 +1,6 @@
 package dev.watchwolf.entities;
 
 public enum ServerType {
```

Comment:
By definition in the WatchWolf Standard, the server type is UpperCammelCase ([ref](https://github.com/watch-wolf/WatchWolf/blob/f941d2f6c97649971eb2757781d6a2c8c1e9bdf3/Standard/Protocols.pdf), in subsection 2.4.7 Server type, under page 18). When you send the 'start server operation' to WatchWolf ServersManager ([dev.watchwolf.tester.TesterConnector, line 237](https://github.com/rogermiranda1000/WatchWolf-Tester/blob/629f3155c00acad5c3d41c8cd80e5b14d688da8a/src/main/java/dev/watchwolf/tester/TesterConnector.java#L237)) you'll have to convert `mcType.name()` to UpperCammelCase.